### ISSUE TITLE
The importance of the warmup stage

### ISSUE THREAD

**Issue by shashacks** [microsoft/GUI-Actor#29]
Hello, I have a question regarding the warm-up stage for model performance.
How critical is the warm-up stage for achieving good performance?
Based on your experiments, is the performance significantly degraded or not feasible if the warm-up stage is skipped?

It seems that during the warm-up stage, you used a larger learning rate for the appended head and a smaller one for full fine-tuning.
I’m curious — what were the results when training was done without a warm-up stage and only full fine-tuning with the smaller learning rate?
